domains/ss.kleb.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "KrazyKleb",
+        "email": "krazykleb@outlook.com"
+    },
+    "record": {
+        "A": "128.254.225.56"
```

```suggestion
        "A": ["128.254.225.56"]
```
Invalid A record format